intro-HPC/ch_x2go.tex

```diff
@@ -0,0 +1,154 @@
+Please see \autoref{sec:using-ssh-agent-with-openssh} if you want to know how to setup an SSH agent in your system.
+
+
+\section{Create a new X2Go session}
+\label{sec:sessions-x2go}
+
+After the X2Go client installation just start the client.
+When you launch the client for the first time, it will start the new session dialogue automatically.
+
+There are two ways to connect to the login node:
+\begin{itemize}
+\item \emph{Option A}: A direct connection to ``\strong{\emph{\loginnode}}''. This is the simpler option, the system will decide which
+login node to use based on a load-balancing algorithm.
+\item \emph{Option B}: You can use the node ``\strong{\emph{\loginnode}}'' as SSH proxy to connect to a specific login node. Use this option if you want to resume an old X2Go session.
+\end{itemize}
+
+\subsection{Option A: direct connection}
+\label{subsec:sessions-x2go-optiona}
+This is the easier way to setup X2Go, a direct connection to the login node.
+
+\begin{center}
+\includegraphics*[width=3.30in, height=3.17in, keepaspectratio=false]{ch19-x2go-configuration-gent}
+\end{center}
+
+\begin{enumerate}
+    \item  Include a session name. This will help you to identify 
+    the session if you have more than one, you can choose any name (in our example ``HPC login node``).
+    \item  Set the login hostname (In our case: ``\strong{\emph{\loginnode}}'')
+    \item  Set the Login name. In the example is ``\strong{\userid}'' but you must change it by your
+    current vsc account.
+    \item  Set the SSH port (22 by default).
+    \item  Skip this step if you are using an SSH agent (see \autoref{sec:x2go-client}).
+    If not add your SSH private key into ``Use RSA/DSA key..'' field. In this case:
+    \begin{enumerate}
+     \item Click on the ``Use RSA/DSA..'' folder icon. This will open a file browser. 
+     \begin{center}
+       \includegraphics*[width=3.2in, keepaspectratio=true]{ch19-x2go-ssh-key}
+     \end{center}
+     \item
+     \ifmacORlinux
+     You should look for your \strong{private} SSH key generated in \autoref{sec:generate-key-pair-with-openssh}.
+     This file has been stored in the directory ``\emph{\tilde/.ssh/}'' (by default ``\strong{id\_rsa}''.
+     `.ssh'' is an invisible directory, the Finder will
+      not show it by default. The easiest way to access the folder, is by pressing
+      \keys{cmd} + \keys{shift} + \keys{g}, which will allow you to enter the name of a directory, which
+     you would like to open in Finder. Here, type ``\tilde/.ssh'' and press enter.
+     Choose that file and click on \keys{open}.
+     \fi
+     \ifwindows
+     You should look for your \strong{private} SSH key generated by \keys{puttygen} in \autoref{sec:generate-key-pair}
+     (by default ``\strong{id\_rsa.ppk}''). This file should have been stored in the directory
+     ``C:\textbackslash Users\textbackslash \%USERNAME\%\textbackslash
+     AppData\textbackslash Local\textbackslash PuTTY\textbackslash .ssh''.
```

Comment:
@alvarosimon I would remove this `This file should have been stored in ...` sentence.

The user should know where they have stored their private key, it could be anywhere, and suggesting them a specific path will only lead to confusion?

You can point to the section where the location is specified in `PuTTY` though, so they can figure out where it's located if they can't remember.

Besides, the `C:\\...` path isn't rendering properly in the PDF version...